fiona/crs.py

```diff
@@ -64,7 +64,7 @@ def from_epsg(code):
     """
     if int(code) <= 0:
         raise ValueError("EPSG codes are positive integers")
-    return {'init': "epsg:%s" % code, 'no_defs': True}
+    return {"init": f"epsg:{code}", "no_defs": True}
```

Comment:
@mwtoews for the first time I realize that we have a crs Cython module and a crs Python module in the master branch! Let's not do anything about that here, this is a note to fix the situation in another PR.

Reply:
This change is removed and issue raised at #1226